doc/rst/source/gmt.conf.rst

```diff
@@ -168,8 +168,9 @@ FONT Parameters
 
     **FONT_LOGO**
         Font to use for text plotted as part of the GMT time logo [:doc:`theme
-        dependent <theme-settings>`]. Choose **auto** for :ref:`automatic scaling
-        with plot size <auto-scaling>`.
+        dependent <theme-settings>`]. **Note**: Since the time logo has a fixed height
```

Comment:
`FONT_LOGO` is not theme-dependent (scaling font size), so we should also remove it from the `theme-settings.rst` file

Reply:
See my commit